app/src/routes/family/drive/$driveId/form/general-info.tsx

```
@@ -0,0 +1,321 @@
+              type="button" 
+              onClick={handleBack} 
+              variant="outline" 
+              className="flex-1 h-14 rounded-xl border-2 border-[var(--color-kfk-blue)] text-[var(--color-kfk-blue)] font-bold text-lg"
+            >
+              <ChevronLeftIcon className="mr-2 h-6 w-6" />
+              Back
+            </Button>
+            
+            <form.Subscribe
+              selector={(state) => [state.canSubmit, state.isSubmitting]}
+              children={([canSubmit, isSubmitting]) => (
```

Biome (2.4.4)

[error] Avoid passing children using a prop

(lint/correctness/noChildrenProp)